setup.py

```diff
@@ -16,18 +16,20 @@ def get_version():
 
 
 setup(
-    name="bmap-tools",
+    name="bmaptool",
     description="Tools to generate block map (AKA bmap) and copy images " "using bmap",
     author="Artem Bityutskiy",
     author_email="artem.bityutskiy@linux.intel.com",
+    maintainer="Trevor Woerner",
+    maintainer_email="twoerner@gmail.com",
```

Comment:
Are you planning to publish to pypi.org?

Reply:
I would really like to do that when I have a little bit of time to actually do it. It's pretty easy these days

Reply:
I think GitHub Actions makes that fairly doable as well, I just haven't done it already.

Reply:
I didn't realize that's what I was signing up to do. I'm just doing a global search and fixup for changing the maintainership. Isn't setup.py a generic, non-pypi-specific, python setup mechanism?

Reply:
It is, but it's deprecated. Although you can be listed as the maintainer without being the person who publishes to pypi, so I wouldn't worry about it too much. This is fine IMHO, and we'll worry about pypi later

Reply:
Agreed. I was just asking to make sure I knew what expectations to have. We'll worry about pypi later.